UI/QuickMenu/ReMenuCategory.cs

```diff
@@ -178,6 +178,12 @@ public ReMenuButton AddButton(string text, string tooltip, Action onClick, Sprit
             return button;
         }
 
+        public ReMenuButton AddSpacer(Sprite sprite = null) {
+            var spacer = new ReMenuButton(string.Empty, string.Empty, null, _buttonContainer.RectTransform, sprite);
```

Comment:
Please give the spacer a name. Also, implement it the same way as the ReMenuPage implementation.